### ISSUE TITLE
<App/> should be set (plugin: plugin-ts, hook: buildStart) (plugin: plugin-ts, hook: buildStart)

### ISSUE THREAD

**Issue by burt202** [kubb-labs/kubb#1393]
### What version of `kubb` is running?

3.0.4

### What platform is your computer?

Linux

### What version of external packages are you using(`@tanstack-query`, `MSW`, `React`, `Vue`, ...)

See linked repo below

### What steps can reproduce the bug?

Maybe similar to https://github.com/kubb-labs/kubb/issues/1379 but Im getting it when using `3.0.4`. For repro, clone repo linked below and `npm ci` and `npm run generate`

https://github.com/burt202/kubb-test/tree/master

Basically im trying to upgrade our setup to v3, which takes a yml openapi docs link and spits out generated TS for it. Im expecting the `generated/` folder to contain the following on successful run:

```
index.ts with 'export * from "./types"'
types.ts with geneated types from kubb
```

Any help would be appreciated as to what im doing wrong. Cheers!

### How often does this bug happen?

Every time

**Comment by stijnvanhulle** [comment]
`v3.0.5` will resolve this, having a wider version for our peerDependency `@kubb/react` resolved the issue.

**Comment by burt202** [comment]
Thanks for the quick response. `v3.0.5` did fix the error but im not getting an `index.ts` generated with the contents `export * from "./types"`. I thought the top level `output.barrelType: 'all',` would achieve this. Any pointers would be appreciated. Cheers!

**Comment by burt202** [comment]
Great thanks. `3.0.5` fixed the issue

**Comment by catesandrew** [comment]
I wasn't even using the @kubb/react but hit this bug. so adding @kubb/react finally fixed it, even if you're using it or not.
